nbextensions/appCell2/widgets/tabs/logTab.js

```diff
@@ -0,0 +1,92 @@
+/**
+ * This is the entrypoint module for the job status / log viewer tab of the app cell.
+ */
+define(['bluebird', 'common/html', 'common/ui', 'util/jobLogViewer'], (
+    Promise,
+    html,
+    UI,
+    JobLogViewerModule
+) => {
+    'use strict';
+
+    const { JobLogViewer } = JobLogViewerModule;
+
+    const t = html.tag,
+        div = t('div'),
+        p = t('p');
+
+    const BATCH_MODE_TEXT =
+        'This app is running in batch mode. The status and log of the parent job are shown below.';
+
+    /**
+     *
+     * @param {object} config with key 'model', containing the cell model
+     * @returns log tab instance
+     */
+    function factory(config) {
+        let container, // The top level node used by this widget
+            ui; // UI module interface to this container
+        const { model } = config,
+            widgets = {};
+
+        function generateLayout(isBatchJob = false) {
+            let intro = '';
+            if (isBatchJob) {
+                intro = p(BATCH_MODE_TEXT);
+            }
+            return intro + div({ dataElement: 'log' });
+        }
```

Comment:
Changed the layout for batch mode in the app cell to be identical to single job mode but with a line at the top saying the content of `BATCH_MODE_TEXT`. It is not possible to see the info (status, params, etc.) for batch mode jobs whilst they are running so there's no point in having the params/log widgets suggesting it is possible.